lib/assets/javascripts/views/phases/edit.js

```diff
@@ -1,7 +1,14 @@
+import expandCollapseAll from '../../utils/expandCollapseAll';
+
+$(() => {
+  // Attach handlers for the expand/collapse all accordions
+  expandCollapseAll();
```

Comment:
As the module expandCollapseAll always walks through any accordion-controls class, i.e. the selector is hardcoded in the module, perhaps is better if we rename this filename to something more generic since it will be used in multiple views, not only at phases/edit.